docs-site/src/env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="astro/client" />
```

Comment:
Is this file necessary if the only line in it is commented out?

Reply:
yes, this is a special comment that enables astro/client types available in the file 👍